src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -79,6 +79,13 @@ private static Option2<T> CreateOption<T>(OptionGroup group, string name, T defa
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_style_pattern_matching_over_is_with_cast_check"),
                 new RoamingProfileStorageLocation($"TextEditor.CSharp.Specific.{nameof(PreferPatternMatchingOverIsWithCastCheck)}")});
 
+        public static readonly Option2<CodeStyleOption2<bool>> PreferNotPattern = CreateOption(
+            CSharpCodeStyleOptionGroups.PatternMatching, nameof(PreferNotPattern),
+            defaultValue: s_trueWithSuggestionEnforcement,
+            storageLocations: new OptionStorageLocation2[] {
+                EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_style_not_pattern"),
```

Comment:
It's surprising to me that editor config toggles are buried in here. Should probably add this to the docs page at least though.